The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/10231295201/job/28307087312) [(plain)](https://github.com/rust-lang/rust/commit/68451857230a9f0aaa485ce8a32314be079e9b0a/checks/28307087312/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
#16 2.663 Building wheels for collected packages: reuse
#16 2.664   Building wheel for reuse (pyproject.toml): started
#16 2.909   Building wheel for reuse (pyproject.toml): finished with status 'done'
#16 2.909   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132715 sha256=dfa09868353292d98f811d3efdb0d54d07389e808efc71d68e3b93c514bf8bec
#16 2.910   Stored in directory: /tmp/pip-ephem-wheel-cache-r0i_8b0a/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#16 2.912 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#16 3.301 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#16 3.301 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#16 DONE 3.4s
---
   Compiling std v0.0.0 (/checkout/library/std)
error[E0711]: feature `slice_first_last_chunk` is declared stable since 1.80.0, but was previously declared stable since 1.77.0
   --> library/core/src/slice/mod.rs:525:5
    |
525 |     #[rustc_const_stable(feature = "slice_first_last_chunk", since = "1.80.0")]

For more information about this error, try `rustc --explain E0711`.
error: could not compile `core` (lib) due to 1 previous error
Build completed unsuccessfully in 0:00:24
```

</details>

--- a/library/core/src/slice/mod.rs
+++ b/library/core/src/slice/mod.rs
@@ -522,7 +522,7 @@ impl<T> [T] {
     /// ```
     #[inline]
     #[stable(feature = "slice_first_last_chunk", since = "1.77.0")]
-    #[rustc_const_stable(feature = "slice_first_last_chunk", since = "1.77.0")]
+    #[rustc_const_stable(feature = "const_slice_last_chunk", since = "1.80.0")]
     pub const fn last_chunk<const N: usize>(&self) -> Option<&[T; N]> {
         if self.len() < N {
             None

